### PR TITLE
Multiaddress instructions added

### PR DESCRIPTION
**Pull request by har00ga**
This PR adds instructions for multiaddresses to the mining page.

### CODE REVIEW

**Comment by har00ga** [comment]
@mishmosh ready for review

**Comment by whyrusleeping** [comment]
@har00ga thanks for this! I think its worth adding a couple sentences about how the addresses you pass to set-addrs should be addresses you are publically dialable on (and to confirm that you are dialable at that address before setting it),  the ListenAddresses you set in your config.toml should be interface listen addresses (usually /ip4/0.0.0.0/tcp/PORT), and the "AnnounceAddresses" you set in the config.toml should match what youre going to call `set-addrs` on.

Honestly, this could be a lot easier, but the whole 'discover your addresses' bit is hard to automate.

File: documentation/en/mining.md [hunk 1]
```diff
@@ -99,3 +99,23 @@ This env var can be used with `lotus-storage-miner`, `lotus-seal-worker`, and `l
 ### `FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1` Environment variable
 
 This env var can be used with `lotus-storage-miner`, `lotus-seal-worker`, and `lotus-bench` to enable experimental precommit2 GPU acceleration
+
+### Setting multiaddresses
+
+It it possible to set multiaddresses for the miner to listen on in a miner's `config.toml` file
```
Comment (mishmosh) [inline documentation/en/mining.md:105]: I would be even more pushy about this step, ie 

"To improve ____ it is recommended to set multiaddresses..." or even (default everyone to this step) "Next, set multiaddresses...."

Comment (har00ga) [reply documentation/en/mining.md]: Adjusted!

**Pull request merged**